workshops/11-Docker-Workshop/README.md

```diff
@@ -0,0 +1,137 @@
+# Docker
```

Comment:
Maybe link to the API references for dockerfiles/docker-compose? 

https://docs.docker.com/engine/reference/builder/

https://docs.docker.com/compose/compose-file/